articles/libraries/auth0-swift/index.md

```diff
@@ -182,9 +228,31 @@ Auth0
    }
 ```
 
+::: panel-info Unlinking - Metadata
+Note that when accounts are linked, the secondary account's metadata is not merged with the primary account's metadata. Similarly, when unlinking two accounts, the secondary account does not retain the primary account's metadata when it becomes separate again.
+:::
+
+#### Update user metadata
+
+When updating user metadata, you will create a `userMetadata` object, and then call the `patch` method, passing it the user id and the `userMetadata` object. The values in this object will overwrite existing values with the same key, or add new ones for those that don't yet exist in the user metadata.
+
+```swift
+Auth0
+    .users(token: "user token")
+    .patch("user identifier", userMetadata: ["first_name": "John", "last_name": "Doe"])
+    .start { result in
+        switch result {
+        case .success(let userInfo):
+            print("user: \(userInfo)")
+        case .failure(let error):
+            print(error)
+        }
+    }
+```
+
 ### Web-based Auth (iOS Only)
 
-First go to [Auth0 Dashboard](https://manage.auth0.com/#/applications) and go to application's settings. Make sure you have in *Allowed Callback URLs* a URL with the following format:
+First go to [Auth0 Dashboard](${manage_url}/#/applications) and go to application's settings. Make sure you have in *Allowed Callback URLs* a URL with the following format:
```

Comment:
@jeffreylees `${manage_url}/#/applications` should be `${manage_url}/#/clients` and `go to application's settings` should be `go to client's settings`.